archetype/src/main/resources/archetype-resources/pom.xml

```diff
@@ -30,7 +30,7 @@
         <dependency>
             <groupId>org.seleniumhq.selenium</groupId>
             <artifactId>selenium-server</artifactId>
-            <version>2.47.1</version>
+            <version>2.48.2</version>
```

Comment:
We probably will have a problem here. The archetype does not use the project BOM and thus both servlet-api 3.1 and 2.5 will be present for projects that use it. We should add the exclusion here